test/views/declaration/spec/UnitViewSpec.scala

```diff
@@ -43,6 +47,36 @@ class UnitViewSpec extends UnitSpec with ViewMatchers {
   }
 
   def messagesKey(key: String): BeMatcher[String] = new MessagesKeyMatcher(key)
```

Comment:
Same here. It is a different pattern of what we are used to.
It would be good to understand the benefits.